### PR TITLE
chore: initial structure

### PR DESCRIPTION
**Pull request by jens-peterolsson**
Two things to consider:

1) Do we want to use the GetCommand or just use the getItem function on the document client? Not really a fan of the send syntax in this context after working with it for a while, but both AWS mocking module and the community in general seems to favour that over the direct functions which seems to be regarded as kept only for legacy/compatibility reasons.

2) Is the exporting mechanism in index.js a good way to structure the lib? Thinking it could be good to hide the instantiation of the document client etc. from clients of this lib and also has the advantage that we only init one instance of the client per lambda instance, but the syntax is a little clumsy and intellisense etc. gets harder when passing in the instance from the calling function.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
-const scan = require('./dynamodb/scan');
+const { DynamoDB } = require('@aws-sdk/client-dynamodb');
+const { DynamoDBDocument } = require('@aws-sdk/lib-dynamodb');
 
-module.exports = {
-  scan
+const getItem = require('./dynamodb/get-item');
+
+const client = new DynamoDB({});
+const documentClient = DynamoDBDocument.from(client);
```
Comment (Crafoord) [inline index.js:7]: There might be a case where you want to send in `options` to this client. Not sure how we would do that with this structure 🤔

Comment (jens-peterolsson) [reply index.js:7]: Right, we should probably make a decision on that so we don't have to change the interface later. One option is to expose a function instead from index.js where you can pass in an options object (or not) and that function could return the other functions (but not the actual client object). If so, we should default to the same instance if options are blank or the same.

**Comment by Crafoord** [comment]
Regarding `1.` I think we should try to follow the community and try to stay mainstream©️. Also here I wonder if we will ever need to pass some extra options to the different commands? 🕵️

**Comment by jens-peterolsson** [comment]
I will merge this PR to avoid multiple iterations on it, and also as the name indicates it's just meant to be the initial structure. Will take the possibility to pass options into consideration and create a new PR with a revised structure.

**Pull request merged**